test/e2e/util.go

```diff
@@ -2041,6 +2042,25 @@ func restartKubeProxy(host string) error {
 	if !providerIs("gce", "gke", "aws") {
 		return fmt.Errorf("unsupported provider: %s", testContext.Provider)
 	}
+	err1 := restartKubeProxyByInitD(host)
```

Comment:
Can we gate this on serverVersion instead?  See:

https://github.com/kubernetes/kubernetes/commit/a5d895ac49d5047f350f991de65076999b7e9d8f#diff-8349006db2c242fd7424e1dfb3295840R959